### PR TITLE
1st localisation crusade

### PR DESCRIPTION
**Pull request by arithon**
**And so it begins again**

## Changelog:
- Renaming some titles (german)
- Working on traits file (german)
- Finished localization in the religions file (german)
- Fixed a small bug with the german localization of MaA
- Finished translation of the core tenets file (german)
- Finished translation of the magical doctrines file (german)
- Translated changes made to the English localization (german)

- Changed one cultural adjective in the replacement file (English)
- Localized the Religion game concept description (game_concept_religion_desc)
- Localized the Religious Family game concept description (game_concept_religious_family_desc)
- Localized the Knight game concept description (game_concept_knight_desc)
- Localized heresy pop up game setting (setting_default_regional_heresy_desc)
- Localized the Organized game concept description (game_concept_organized_desc)
- Localized innovations (innovation_development_03_desc & innovation_cranes_desc)
- Localized Dynasty, Last unlock in the Blood Legacy tree (blood_legacy_5_name)
- Localized Praetorian Guard and Parthian Tactics perk names
- Many changes based on Issue 101: https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/issues/101
- Literally this for all the words that are different between the two dialects: 
image

## How to test:
German:
- Check the different religions, tenets and doctrines in-game. You'd have to switch faiths or persons to see all of the doctrines because for example shaman or goblin tenets tend not to appear when you're of another faith group.
- Check the changes that got made to the English localization, too.
- (You can simply only read the changes in the file, too, if that's easier for you, but when new to ck3 localization code, it might help to check how it plays out in-game.)

English:
- Check the changes that got made to the English localization.
- (You can simply only read the changes in the file, too, if that's easier for you, but when new to ck3 localization code, it might help to check how it plays out in-game.)

### CODE REVIEW

**Review by Gaben1025 — APPROVED** [review]

File: localization/english/modifiers/wc_modifiers_l_english.yml [hunk 1]
```diff
@@ -7,25 +7,34 @@
...
- massive_infantry_siege_value_add:1 "[massive_infantry|E] [siege_progress|E]"
  massive_infantry_damage_add:1 "[massive_infantry|E] [damage|E]"
  massive_infantry_damage_mult:0 "$massive_infantry_damage_add$"
  massive_infantry_toughness_add:1 "[massive_infantry|E] [toughness|E]"
  massive_infantry_toughness_mult:0 "$massive_infantry_toughness_add$"
+ massive_infantry_pursuit_add:1 "[massive_infantry|E] [pursuit|E]"
+ massive_infantry_pursuit_mult:0 "$massive_infantry_pursuit_add$"
  massive_infantry_screen_add:1 "[massive_infantry|E] [screen|E]"
+ massive_infantry_screen_mult:0 "$massive_infantry_screen_add$"
+ massive_infantry_siege_value_add:1 "[massive_infantry|E] [siege_progress|E]"
```
Comment (arithon) [inline localization/english/modifiers/wc_modifiers_l_english.yml:40]: You should avoid inserting lines in between others, new lines are supposed to always be added at the end.

Comment (Gaben1025) [reply localization/english/modifiers/wc_modifiers_l_english.yml:40]: Dont worry, I made this file in my last branch and I'm okay with this layout.  Looks like I missed some modifier localization and I added it in its appropriate spot.

**Review by Gaben1025 — CHANGES_REQUESTED** [review]
Epidemic will be removed as a tech and added as a religious tenant, you can save the description as I did, because I will most likely use the same one. Besides that you should delete that translation from the branch.

**Review by Gaben1025 — APPROVED** [review]

**Review by Erkor001 — APPROVED** [review]

File: localization/english/wc_traits_l_english.yml [hunk 2]
```diff
@@ -24,7 +24,7 @@
...
  trait_creature_naga_character_desc:0 "$trait_creature_naga_desc$"
  trait_creature_blood_elf:0 "Blood Elf"
- trait_creature_blood_elf_desc:0 "The blood elves are a race formed from the high elves of Quel'Thalas. In honour of their dead brethren, the survivors call themselves blood elves or Sin'dorei, which in thalassian means "children of blood". Both high elves and blood elves, regardless of class, status or origin, are fond of magic and are somewhat addicted to it. On average, blood elves are slightly taller than humans and considerably thinner in bone structure."
+ trait_creature_blood_elf_desc:0 "The blood elves are a race formed from the high elves of Quel'Thalas. In honor of their dead brethren, the survivors call themselves blood elves or Sin'dorei, which in thalassian means "children of blood". Both high elves and blood elves, regardless of class, status or origin, are fond of magic and are somewhat addicted to it. On average, blood elves are slightly taller than humans and considerably thinner in bone structure."
```
Comment (LordSilvermane) [inline localization/english/wc_traits_l_english.yml:27]:
```suggestion
 trait_creature_blood_elf_desc:0 "The blood elves are a race formed from the high elves of Quel'Thalas. In honor of their dead brethren, the survivors call themselves blood elves or Sin'dorei, which in Thalassian means "children of blood". Both high elves and blood elves, regardless of class, status or origin, are fond of magic and are somewhat addicted to it. On average, blood elves are slightly taller than humans and considerably thinner in bone structure."
```

**Review by LordSilvermane — COMMENTED** [review]
Found only one english problem, thalassian should be capitalized.

Did want to bring one thing up I wasn't sure of, though. Elunic religion has Ancient Guardians capitalized, while Ursine does not. Is there a reason for the difference, or was it overlooked?

File: localization/english/wc_traits_l_english.yml (hunk 2 above)
Comment (arithon) [reply localization/english/wc_traits_l_english.yml:27]: Oh, good point. For some reason I thought of the adjective here, but you're right, it's the language like English/German etc.

**Review by LordSilvermane — APPROVED** [review]
LGTM!

File: localization/german/culture/wc_cultural_innovations_l_german.yml [hunk 3]
```diff
@@ -58,4 +58,4 @@
  innovation_watersmithing_desc:0 "Im Laufe der Äonen haben wir gelernt, Wasser zu manipulieren, um außergewöhnliche Bauwerke aus Fels und Stein zu errichten, die jedem Vergleich standhalten. Unsere Waffen und Rüstungen sind so hart wie jeder geschmiedete Stahl, und unsere Mauern scheinen oft fugenlos konstruiert zu sein."
```
Comment (panpoleon) [inline localization/german/culture/wc_cultural_innovations_l_german.yml:58]: "Im Laufe der Äonen haben wir gelernt Wasser zu manipulieren" kein Komma zwischen gelernt und Wasser

File: localization/replace/german/wc_game_concepts_l_german.yml [hunk 4]
```diff
@@ -1,9 +1,16 @@
...
-     game_concept_spearmen_desc:0 "Speerträger sind [men_at_arms|E], das zu Fuß mit Stangenwaffen kämpft.\n\nAuch wenn alle Speerträger einzigartig sind, [Concept('counter', 'Kontern')|E] sie meist [heavy_cavalry|E], [light_cavalry|E] und [massive_infantry|E]und werden wiederum von [heavy_infantry|E] gekontert.\n\nSpeerträger sind oft auch stark in hügeligem und gebirgigem [terrain|E]."
-     game_concept_heavy_infantry_desc:0 "Schweres Fußvolk ist [men_at_arms|E], das zu Fuß mit schweren Rüstungen und Waffen kämpft.\n\nAuch wenn alles Schwere Fußvolk einzigartig ist, [Concept('counter', 'Kontert')|E] es meist [spearmen|E] und wird wiederum von [skirmishers|E] gekontert."
-     
\ No newline at end of file
+﻿l_german:
+ #DESCRIPTION FOR UNITS
+ game_concept_skirmishers_desc:0 "Plänkler sind [men_at_arms|E], das leicht bewaffnet und geübt im Drangsalieren des Gegners ist.\n\nAuch wenn alle Plänkler einzigartig sind, [Concept('counter', 'Kontern')|E] sie meist [heavy_infantry|E] und [caster|E] und werden wiederum von [archers|E] und [massive_infantry|E] gekontert."
```
Comment (panpoleon) [inline localization/replace/german/wc_game_concepts_l_german.yml:3]: "Drangsalieren" ist ein Verb -> kleingeschrieben

File: localization/replace/german/wc_game_concepts_l_german.yml [hunk 5]
```diff
@@ -1,9 +1,16 @@
-﻿ l_german:
-     #DESCRIPTION FOR UNITS
-     game_concept_skirmishers_desc:0 "Plänkler sind [men_at_arms|E], das leicht bewaffnet und geübt im Drangsalieren des Gegners ist.\n\nAuch wenn alle Plänkler einzigartig sind, [Concept('counter', 'Kontern')|E] sie meist [heavy_infantry|E] und [caster|E] und werden wiederum von [archers|E] und [massive_infantry|E] gekontert."
```
Comment (panpoleon) [inline localization/replace/german/wc_game_concepts_l_german.yml:3]: "Drangsalieren" ist ein Verb -> kleingeschrieben

**Review by panpoleon — CHANGES_REQUESTED** [review]
Kommafehler bei innovation_watersnuthing_desc0

Groß- und Kleinschreib Fehler bei localization/replace/german/wc_game_concepts_l_german.yml (allerdings bin ich mir dabei nicht zu 100% sicher)

File: localization/replace/german/wc_game_concepts_l_german.yml (hunk 4 above)
Comment (arithon) [reply localization/replace/german/wc_game_concepts_l_german.yml:3]: Ne, ist so richtig.

File: localization/german/culture/wc_cultural_innovations_l_german.yml (hunk 3 above)
Comment (arithon) [reply localization/german/culture/wc_cultural_innovations_l_german.yml]: Erledigt.

**Review by panpoleon — APPROVED** [review]

**Pull request merged**